### DISCUSSION TITLE
Pdf won't turn dark

### DISCUSSION THREAD

**Discussion by LukeHeffAI** [tefkah/zotero-night#149] [Q&A] [answered]
Hi all,

Night mode seems to work just fine in the libraries section, but the files themselves won't seem to go dark. Does anyone know a fix for this? See screenshots for Soothing Library vs Blinding Papers.

Using Zotero 6.0.26 on Ubuntu 22.04.3. I've tried the basics of switching it off and on + uninstalling and reinstalling.

![image](https://github.com/tefkah/zotero-night/assets/125118901/455dadf6-742c-4bb7-9664-37a837280e85)

![image](https://github.com/tefkah/zotero-night/assets/125118901/d63c19d9-61cb-40eb-9179-b128cd83f028)

**Comment by tefkah** [comment, marked as answer]
Have you tried pressing this little guy?

![Screenshot_20230904-120912_Brave.jpg](https://github.com/tefkah/zotero-night/assets/21983833/ada40089-de7f-4a27-8a64-0845848f2b98)

**Reply by LukeHeffAI** [reply]
You're a genius and a lifesaver.
